File: src/WritingGoods.java

public class WritingGoods extends Goods {
	private String material;
    GoodsType goodsType;

    public WritingGoods(String name, double price, int amount, ColorType color, GoodsType goodsType, CustumerType grade, String material) {
    	super(name, price, amount, color, grade); 	
    	setMaterial(material);
    	setGoodsType(goodsType);
    }
	public String getMaterial() {
		return material;
	}
	public void setMaterial(String material) {
		this.material = material;
	}
	public GoodsType getGoodsType() {
		return goodsType;
	}
	public void setGoodsType (GoodsType goodsType) {
		this.goodsType = goodsType;
	}
	@Override
	public String toString() {
		return super.toString() + " material = " + material + ", goodsType = " + goodsType + "\n";
	}
 
	
	
}

File: python/GShop/Goods/WritingGoods.py
from Goods.Goods import Goods
from enums.GoodsType import GoodsType



class WritingGoods(Goods):
    goods_type = GoodsType

    def __init__(self, name, price, amount, custumer_type, color, goods_type):
        self.goods_type = goods_type

        Goods.__init__(self, name, price, amount, custumer_type, color)

    def __str__(self):
        return "Name : " + str(self.name) + " price: " + str(self.price) + "amount: " + str(self.amount) \
               + "color: " + str(self.color) + "custumer type: " + str(self.custumer_type)